regulations/static/regulations/css/less/fonts.less

```diff
@@ -6,45 +6,45 @@
 
 @font-face{
     font-family: "Avenir Next";
-    src: url("../../eregsip/font/e9167238-3b3f-4813-a04a-a384394eed42.eot");
-    src: url("../../eregsip/font/e9167238-3b3f-4813-a04a-a384394eed42.eot?iefix") format("embedded-opentype"),
-         url("../../eregsip/font/1e9892c0-6927-4412-9874-1b82801ba47a.woff") format("woff");
+    src: url("../../fonts/e9167238-3b3f-4813-a04a-a384394eed42.eot");
+    src: url("../../fonts/e9167238-3b3f-4813-a04a-a384394eed42.eot?iefix") format("embedded-opentype"),
```

Comment:
You can drop everything but woff and woff2 files. See https://github.cfpb.gov/CFPB/hubcap/issues/66

Reply:
See https://github.com/cfpb/capital-framework/pull/697 for example.